package.json

```diff
@@ -75,6 +75,7 @@
     "electron-builder": "^13.9.0",
     "electron-builder-squirrel-windows": "^12.3.0",
     "electron-packager": "^8.5.2",
+    "electron-rebuild": "^1.5.7",
```

Comment:
I've moved this dependency to dev-dependencies as recommended by electron-builder